index.html

```diff
@@ -29,7 +29,7 @@
 				</div>
 				<div class="list-section">
 					<div class="add-task">
-						<p class="task-title">Tasks</p>
+						<h2 class="task-title">Tasks</h2>
```

Comment:
Same comment as the other one. You still need an h1 before you even think about using an h2.